packages/radio-group/src/vaadin-radio-button.js

```diff
@@ -78,200 +103,62 @@ class RadioButtonElement extends ElementMixin(ControlStateMixin(ThemableMixin(Ge
           cursor: inherit;
           margin: 0;
         }
-
-        :host([disabled]) {
-          -webkit-tap-highlight-color: transparent;
-        }
       </style>
-
-      <label>
-        <span part="radio">
-          <input
-            type="radio"
-            checked="[[checked]]"
-            disabled$="[[disabled]]"
-            role="presentation"
-            on-change="_onChange"
-            tabindex="-1"
-          />
-        </span>
-
-        <span part="label">
-          <slot></slot>
-        </span>
-      </label>
+      <div part="container">
+        <div part="radio">
+          <slot name="input"></slot>
```

Comment:
The tabbing behaviour has a regression since the buttons don't share a unique name inside one group:

https://user-images.githubusercontent.com/1222264/134319237-321982dd-2584-4c91-bdb0-8e8fe8229398.mp4

Reply:
Oh, well, that's a problem. First, thanks for the good catch!

The reason it behaves so is in lack of the `name` attribute on the radio buttons. 

Maybe we should consider auto-generating a single name for the radio buttons inside a group.

Reply:
Fixed by setting a common name attribute for the radio buttons within a radio group.